mobify/platform-scaffold
# [WEB-1369] Delete Hardcoded References to Merlin's Potions

Audit scaffold codebase for hard coded features to remove

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1369

## Changes
- Remove merlins specific hardcoded text 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- go through the site and check if there is no merlin's specific code
